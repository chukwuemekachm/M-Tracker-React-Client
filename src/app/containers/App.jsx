import React from 'react';
import HomePage from './HomePage';
import '../assets/css/style.css';

const App = () => (
  <div>
    <HomePage />
  </div>
);

export default App;
