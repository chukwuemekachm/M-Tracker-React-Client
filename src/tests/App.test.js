import React from 'react';
import { shallow } from 'enzyme';
import App from '../app/containers/App';

describe('App component', () => {
  it('should render without errors', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
