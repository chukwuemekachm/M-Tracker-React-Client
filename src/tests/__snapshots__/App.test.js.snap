// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render without errors 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Welcome to Maintenace Tracker React Application
      </h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Welcome to Maintenace Tracker React Application",
      },
      "ref": null,
      "rendered": "Welcome to Maintenace Tracker React Application",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Welcome to Maintenace Tracker React Application
        </h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Welcome to Maintenace Tracker React Application",
        },
        "ref": null,
        "rendered": "Welcome to Maintenace Tracker React Application",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
